packages/transaction-controller/src/TransactionController.test.ts

```diff
@@ -399,8 +401,9 @@ const TRANSACTION_META_MOCK = {
   time: 123456789,
   txParams: {
     from: ACCOUNT_MOCK,
+    to: ACCOUNT_2_MOCK,
   },
-} as TransactionMeta;
+} as any as TransactionMeta;
```

Comment:
This doesn't seem to be resolving any errors. Maybe try restarting the typescript server in your ide?

Reply:
The typing here can be handled without casting:

```diff
@@ -395,7 +395,9 @@ const ACCOUNT_2_MOCK = '0x08f137f335ea1b8f193b8f6ea92561a60d23a211';
 const NONCE_MOCK = 12;
 const ACTION_ID_MOCK = '123456';
 
-const TRANSACTION_META_MOCK = {
+const TRANSACTION_META_MOCK: TransactionMeta = {
+  id: '1',
+  chainId: '0x1',
   hash: '0x1',
   status: TransactionStatus.confirmed,
   time: 123456789,
@@ -403,16 +405,18 @@ const TRANSACTION_META_MOCK = {
     from: ACCOUNT_MOCK,
     to: ACCOUNT_2_MOCK,
   },
-} as any as TransactionMeta;
+};
 
-const TRANSACTION_META_2_MOCK = {
+const TRANSACTION_META_2_MOCK: TransactionMeta = {
+  id: '2',
+  chainId: '0x1',
   hash: '0x2',
   status: TransactionStatus.confirmed,
   time: 987654321,
   txParams: {
     from: '0x3',
   },
-} as TransactionMeta;
+};
```

Reply:
Done.